Rust/src/common/divector.rs

```diff
@@ -163,4 +163,12 @@ impl DiVector {
         }
     }
 
+    pub fn dimension(&self) -> usize {
```

Comment:
Do we call this `dimensions` (plural) in Java?

Reply:
Yes. Fixed.